python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -296,66 +278,45 @@ async def get_camera_cable_wrap_demand(self):
             Desired camera cable wrap time (TAI unix seconds).
             This will be ``config.camera_cable_wrap_advance_time``
             seconds in the future.
-        """
 
-        rot_data = await self.rotator.tel_rotation.next(flush=True)
-        ccw_data = await self.mtmount_remote.tel_cameraCableWrap.aget()
-
-        # Adjust the camera cable wrap position to the timestamp
-        # in the rotator telemetry.
-        dt = rot_data.timestamp - ccw_data.timestamp
-        ccw_actual_velocity = ccw_data.velocityActual
-        ccw_unadjusted_actual_position = ccw_data.angleActual
-        ccw_actual_position = ccw_actual_velocity * dt + ccw_unadjusted_actual_position
-
-        distance_ccw_rot_actual = ccw_actual_position - rot_data.actualPosition
+        Raises
+        ------
+        asyncio.TimeoutError
+            If data not seen in ROTATOR_TELEMETRY_TIMEOUT seconds.
+        """
 
-        distance_ccw_rot_demand = ccw_actual_position - rot_data.demandPosition
+        rot_data = await self.rotator.tel_rotation.next(
+            flush=True, timeout=ROTATOR_TELEMETRY_TIMEOUT
+        )
 
-        distance_rot_actual_demand = rot_data.actualPosition - rot_data.demandPosition
+        desired_tai = salobj.current_tai() + self.config.camera_cable_wrap_advance_time
+        dt = rot_data.timestamp - desired_tai
 
         if (
-            not self.catch_up_mode
-            and abs(distance_ccw_rot_actual) < self.ccw_rot_sync_limit_slew
-        ) or (
-            abs(distance_ccw_rot_demand) < self.ccw_rot_sync_limit_goto
-            and abs(distance_ccw_rot_actual) < self.ccw_rot_sync_limit_max / 2.0
+            abs(rot_data.demandPosition - rot_data.actualPosition)
+            > self.config.max_rotator_position_error
         ):
-            # Camera cable wrap and camera rotator are synchronized,
-            # or within the goto limit and following closely enough;
-            # follow the demand and exit catch-up mode (if in it).
-            self.catch_up_mode = False
-            desired_position = rot_data.demandPosition
-            desired_velocity = 0.0
+            if not self.rotator_position_error_excessive:
+                self.log.warning("Excessive rotator demand-actual error; using actual")
```

Comment:
I know this is easy to query in the EFD but consider logging the "position error" and  "max_rotator_position_error" when this occurs.

Reply:
OK. I was a bit reluctant because this has to be the Nth sequential excessive error and I was not keen to print all N of them, but printing the last one is better than not printing any of it.